src/hermes/commands/harvest.py

```diff
@@ -29,25 +37,14 @@ def harvest_cff(click_ctx: click.Context, ctx: HermesContext):
         return 1
```

Comment:
Not sure what "the docs" of click say here, but maybe raising an exception is preferrable over return a status code.

(This might also be somehow related to our wrapping `harvest` command... we'll discuss on that maybe)

Reply:
Yes, agree that raising an exception is better. Raising `HermesValidationError` in 2e709b601cae318ec72c93a9b78610a1db3b5cd1. Decided that semantically, this is the correct error to raise (instead of creating a new custom one), as per the API docs.

Currently, these aren't caught anywhere, so I agree that more discussion is necessary. (e.g., try/except in `harvest.py` or elsewhere?)